crates/ruff_python_formatter/tests/snapshots/format@preview.py.snap

```diff
@@ -1,13 +1,45 @@
 ---
 source: crates/ruff_python_formatter/tests/fixtures.rs
-input_file: crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/assign_breaking.py
+input_file: crates/ruff_python_formatter/resources/test/fixtures/ruff/preview.py
```

Comment:
Should we instead run every test under preview and non-preview?

Reply:
`hyperfine "cargo test -p ruff_python_formatter"` goes from 500ms to 580ms on my machine, acceptable for me. 

Having both versions makes it hard to see what actually changed given our large snapshots, so i opted for instead adding the diff as separate code block if there is one. I also changed all non-function/class tests to use `pass` instead as real code generally does, which i split out to #8049 to keep the diff readable.

Reply:
I would like to keep `preview.py` as-is though, it's a useful tool for tracking progress and comparing to black (i can diff both snapshots with what the black playground does)